Would you be open to working with me and an app designer (I'd pay for them) to design a nice ui around your software

ILikeCav opened the pull request:
Would you be open to working with me and an app designer (I'd pay for them) to design a nice ui around your software (btw love the idea and the work your doing)

CalcProgrammer1 committed:
Add Razer Blade Pro 2017 to supported devices under Linux

CalcProgrammer1 committed:
Development branch for using OpenAuraSDK's generic RGBController interface with Keyboard Visualizer

CalcProgrammer1 committed:
Get OpenAuraSDK test branch building on Windows

CalcProgrammer1 committed:
Update OpenAuraSDK

CalcProgrammer1 committed:
Generate effects per zone, single and linear zones supported

CalcProgrammer1 committed:
Multithreaded LED update function

CalcProgrammer1 committed:
Add support for matrix devices in OpenAuraSDK branch, make orange and purple look better

CalcProgrammer1 committed:
Update OpenAuraSDK submodule and build E1.31 support in Linux

CalcProgrammer1 committed:
Update OpenAuraSDK submodule

CalcProgrammer1 committed:
Implement beginnings of OpenRGB SDK support

CalcProgrammer1 committed:
Send all controllers sequentially and let OpenRGB handle threading

CalcProgrammer1 committed:
Remove files that are no longer needed after switching to OpenRGB SDK

CalcProgrammer1 committed:
Treat matrix devices as single zone for now

CalcProgrammer1 committed:
Use tree view for device list and add columns for LED count and zone type

CalcProgrammer1 committed:
Add license (GPLv2)

CalcProgrammer1 committed:
Resizable GUI

CalcProgrammer1 committed:
Matrix effect support

CalcProgrammer1 committed:
Remove visual studio project and get the Qt project to build on Windows.  Remove Arduino sketches, these will be moved elsewhere

CalcProgrammer1 committed:
Use DeviceUpdateLEDs for now so that the threading behavior doesn't corrupt packets

CalcProgrammer1 committed:
Remove unnecessary loop for matrix devices

CalcProgrammer1 committed:
Pull up OpenRGB network_devel branch to pull in nodelay changes

CalcProgrammer1 committed:
Update OpenRGB submodule to use std::thread for client thread

CalcProgrammer1 committed:
Values of 0xFFFFFFFF in matrix map indicate unused slots.  Don't update the colors for these

CalcProgrammer1 committed:
Use std::thread for Keyboard Visualizer threads

CalcProgrammer1 committed:
Add box to enter IP and port of OpenRGB SDK server

CalcProgrammer1 committed:
Allow multiple OpenRGB SDK client connections simultaneously

CalcProgrammer1 committed:
Add support for MacOS

CalcProgrammer1 committed:
Send client string

CalcProgrammer1 committed:
Update OpenRGB submodule to master

CalcProgrammer1 committed:
Automatically reconnect to server if disconnected, display controllers by client connection in list

CalcProgrammer1 committed:
Update client list whenever status changes

CalcProgrammer1 committed:
Update OpenRGB submodule

CalcProgrammer1 committed:
Add button to disconnect clients

CalcProgrammer1 committed:
Code cleanup for version 4.0

CalcProgrammer1 committed:
Update version to 4.00 and move version definition to QMake project

CalcProgrammer1 committed:
Set default delay to 25ms

CalcProgrammer1 committed:
Add functions for creating zone maps and use them to improve zone mapping

CalcProgrammer1 committed:
Store the index map rather than regenerating it every cycle

CalcProgrammer1 committed:
Update OpenRGB submodule

CalcProgrammer1 committed:
Add MacOS icon

CalcProgrammer1 committed:
Update README for OpenRGB change

CalcProgrammer1 committed:
Update README build section

CalcProgrammer1 committed:
Update OpenRGB submodule, update default port, and update client string

CalcProgrammer1 committed:
Update submodule and add GitLab CI configuration to build Windows 32/64 binaries

CalcProgrammer1 committed:
Add KV paths to include path

CalcProgrammer1 committed:
Replace jom with make

CalcProgrammer1 committed:
Fix make path

CalcProgrammer1 committed:
Add no_batch option and switch back to jom

CalcProgrammer1 committed:
Add GitLab CI configuration for project Windows settings

CalcProgrammer1 committed:
Add Debian packaging files

CalcProgrammer1 committed:
Reenable settings file parsing and uncomment strtok_r usage, but use strtok_s in Windows

CalcProgrammer1 committed:
Fix Windows build by making pointer const

CalcProgrammer1 committed:
Add GitLab CI Linux build target that produces an AppImage

CalcProgrammer1 committed:
Make build-appimage.sh executable

CalcProgrammer1 committed:
Fix file name in AppImage script

CalcProgrammer1 committed:
Fix file name in Gitlab CI config

CalcProgrammer1 committed:
Add enable control for each device and initialize devices without a Direct mode to disabled

CalcProgrammer1 committed:
Update linux-deploy path

CalcProgrammer1 committed:
Handle device list updates

MitchRapp01 commented:
would love to see this for the per key led rgb keyboard for the msi line.

On Fri, Jan 18, 2019 at 7:28 PM ILikeCav <notifications@github.com> wrote:

> Would you be open to working with me and an app designer (I'd pay for
> them) to design a nice ui around your software (btw love the idea and the
> work your doing)
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/CalcProgrammer1/KeyboardVisualizer/pull/232
> Commit Summary
>
>    - Add Razer Blade Pro 2017 to supported devices under Linux
>
> File Changes
>
>    - *M* KeyboardVisualizerCommon/RazerChromaLinux.cpp
>    <https://github.com/CalcProgrammer1/KeyboardVisualizer/pull/232/files#diff-0>
>    (7)
>
> Patch Links:
>
>    - https://github.com/CalcProgrammer1/KeyboardVisualizer/pull/232.patch
>    - https://github.com/CalcProgrammer1/KeyboardVisualizer/pull/232.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CalcProgrammer1/KeyboardVisualizer/pull/232>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AWiPA_rtKxtWGzvBLS0cmTAKkhqF1a9Xks5vEmaagaJpZM4aI2mq>
> .
>

SapuSeven committed:
Update icon path in README.md

dd9d9202dd672312ab7f532b73913db4c153bba8 renamed the icon file.

This is a small fix to update the path to the image in the README file.

CalcProgrammer1 committed:
Fix settings file name

dvdvideo1234 commented:
I would love to see you merge my Bulgarian flag pull request.

CalcProgrammer1 committed:
Use C++14 on MacOS

RafaelKr committed:
Fix README steps to build binary on Linux

CalcProgrammer1 committed:
Create FUNDING.yml

CalcProgrammer1 committed:
Update OpenRGB submodule, use bundled AppImage creation tools

CalcProgrammer1 committed:
Add MacOS CI

CalcProgrammer1 committed:
Add 32- and 64-bit Linux builds, AppImage and .deb

CalcProgrammer1 committed:
Remove KeyboardVisualizer network synchronization feature

CalcProgrammer1 committed:
Remove Release submodule, as it only contains 3.x and earlier releases and is no longer relevant for 4.x
